app/src/main/java/com/azuresamples/msalnativeauthandroidkotlinsampleapp/AccessApiFragment.kt

```diff
@@ -31,8 +34,11 @@ class AccessApiFragment : Fragment() {
     companion object {
         private val TAG = AccessApiFragment::class.java.simpleName
         private enum class STATUS { SignedIn, SignedOut }
-        private const val WEB_API_BASE_URL = "" // Developers should set the respective URL of their web API here
-        private val scopes = listOf<String>() // Developers should set the respective scopes of their web API here
+        private const val WEB_API_BASE_URL_1 = "" // Developers should set the first respective URL of their web API resource here
```

Comment:
nit: name this variable `WEB_API_URL_1`. It's not a base URL, it's the full URL.

Same for the URL_2 variant.